docs/install.md

```diff
@@ -61,6 +62,8 @@ Additionally, `make init` creates a symlink called `otterdog` in `~/.local/bin`,
 $ otterdog --version
 ```
 
+Note that if you are building manually on the `main` branch, and not a tag, the version number will be `0.0.0`.
```

Comment:
It's not even that. I just did a clean test:

```
$ git clone git@github.com:eclipse-csi/otterdog.git
$ cd otterdog
$ make init
...
Installing the current project: otterdog (0.0.0)
...
$ ./otterdog.sh --version
otterdog.sh, version 0.0.0
$ make init
...
Installing the current project: otterdog (1.1.0.dev3)
...
$ ./otterdog.sh --version
otterdog.sh, version 1.1.0.dev3
```

so it seems that the dynamic version does not work on first `make init` / `poetry sync`, but then works as expected. @kairoaraujo @netomi, any idea?

Reply:
the problem is that with `poetry sync`, 2 things happen:

- install the necessary plugins, including the dynamic version plugin
- install the dependencies and the project itself in its venv

Now, when poetry is run the first time, the dynamic version plugin is not there yet and thus the version can not be resolved, which leads to the case of `0.0.0`.

You can workaround that as follows in the Makefile:

```
poetry sync --only-root
poetry sync
```

instead of a single `poetry sync`.

Reply:
after the first sync, the plugins are installed, and then the second sync correctly determines the version.

Reply:
I added this change here https://github.com/eclipse-csi/otterdog/pull/395
If this goes first I can rebase, no worries.

Reply:
created https://github.com/eclipse-csi/otterdog/pull/399 to fix this and make subsequent `make init` runs fast.